App.tsx

```diff
@@ -5,22 +5,65 @@
  * @format
  */
 
-import React, { useState, FunctionComponent, useEffect } from "react";
-import { SafeAreaView, StatusBar, View, ViewProps } from "react-native";
-import { EngineView, useEngine } from "@babylonjs/react-native";
-import { SceneLoader } from "@babylonjs/core/Loading/sceneLoader";
-import { Camera } from "@babylonjs/core/Cameras/camera";
-import { ArcRotateCamera } from "@babylonjs/core/Cameras/arcRotateCamera";
-import "@babylonjs/loaders/glTF";
+import React, {
```

Comment:
I guess you are using some formatter (prettier maybe)? The formatting is all different. Maybe not a good idea to change it for this kind of PR.

Reply:
You're right, I missed that, I reverted the formatting changes.